sbin/patchman

```diff
@@ -408,6 +408,18 @@ def dns_checks(host=None):
         host.check_rdns()
 
 
+def host_delete(host=None):
+    """ Look for a host, if found, delete it
+    """
+    if not host:
+        print('No host specificed, doing nothing.')
+        return
```

Comment:
We can probably just delete the `if not host:` section, it doesn't do anything useful.

Reply:
Yep, just verified this on my install, removed.